### PR TITLE
rm: Add documentation for new SBOM feature

### PR DESCRIPTION
**Pull request by doanac**
@kprosise - this is a super rough document. I'm hoping I can get a lot of help from you on making it readable. It's not urgent.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 715e8086c07e6cda59d7036ad4d1479b97cb2b74 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1664/docs/artifacts/html/index.html

File: source/reference-manual/factory/sboms.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+The FoundriesFactory SBOM feature allows users to extract SBOM data and analyze it according to their needs.
+
+Background
+----------
+
+FoundriesFactory CI builds have a unique position to build up SBOMS whenever a change is built in the Factory.
+There are two types of builds that SBOMs are generated for:
+
+ * Yocto builds produce `Software Package Data Exchange`_ (SPDX) artifacts using built-in tooling.
```
Comment (angolini) [inline source/reference-manual/factory/sboms.rst:19]: I would say this in a different way. The build is platform, not yocto. And build-in tooling is from yocto. Something as:

Platform/LmP/Factory builds produce SPDX artifacts using the Yocto Project built-in tooling.

File: source/reference-manual/factory/sboms.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Both builds store artifacts under an ``sboms`` directory.
+These files can be downloaded directly from the web UI when viewing a Target.
+Yocto builds a comprehensive SPDX per image type.
+For example, ``lmp-factory-image-intel-corei7-64.spdx.tar.zst``.
+This archive format includes an SPDX file for every package included in the build.
+Container builds produce an SPDX file per container/architecture such as ``hub.foundries.io/<FACTORY>/shellhttpd/arm64.sdpx.json``.
+
+All this information can be accessed in its raw form by clicking around
+CI builds.
+However, APIs have been built to allow tooling like fioctl to more easily work with Factory SBOMs.
```
Comment (angolini) [inline source/reference-manual/factory/sboms.rst:31]: I would say this with fewer words and without the however in the middle, something like:

"All this information can be accessed in its raw form by clicking around
CI builds or using fioctl to access a more clean/straightforward information with Factory SBOMs"

**Review by angolini — APPROVED** [review]
I think it's a great starting point!
My only concern is using "yocto" to differ from the container targets, because we use another term in factory UI and other documentation (although I understand they are yocto builds)

**Comment by kprosise** [comment]
@doanac I have started to review and suggest edits locally, I will report my feedback here once I have finished.

**Comment by kprosise** [comment]
> I think it's a great starting point!
> 
> My only concern is using "yocto" to differ from the container targets, because we use another term in factory UI and other documentation (although I understand they are yocto builds)

@angolini I agree with all the points in your review and will incorporate them in my review patch.

**Comment by kprosise** [comment]
@doanac I am considering breaking this up into two pages. One for technical explanations, variable lists and background info (reference manual), the other focused on working with them in practice, with a more "how to" feel. (User guide)

**Comment by doanac** [comment]
that sounds like a good idea to me. the current format really forces a lot upon the reader. thanks.

**Comment by kprosise** [comment]
@doanac this is primary focus at the moment. I'm actively working on separating into two pages. As such, this review/edit will take a touch longer than usual. I will keep you up to date.

**Comment by kprosise** [comment]
Closing as content was copy-edited and placed in a separate PR.

**Pull request closed without merging**